EIPS/eip-4824.md

```diff
@@ -0,0 +1,220 @@
+---
+eip: 4824
+title: Decentralized Autonomous Organizations
+description: A standard URI and JSON schema for decentralized autonomous organizations (DAOs).
+author: Joshua Tan (@thelastjosh), Isaac Patka (@ipatka), Ido Gershtein <ido@daostack.io>, Eyal Eithcowich <eyal@deepdao.io>, Michael Zargham (@mzargham), Sam Furter (@nivida)
+discussions-to: https://ethereum-magicians.org/t/eip-4824-decentralized-autonomous-organizations/8362
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-02-17
+---
+
+## Abstract
+
+A standard URI and JSON schema for decentralized autonomous organizations (DAOs), focusing on relating on-chain and off-chain representations of membership and proposals.
+
+## Motivation
+
+DAOs, since being invoked in the Ethereum whitepaper, have been vaguely defined. This has led to a wide range of patterns but little standardization or interoperability between the frameworks and tools that have emerged. Standardization and interoperability are necessary to support a variety of use-cases. In particular, a standard daoURI, similar to tokenURI in [ERC-721](/EIPS/eip-721), will enhance DAO search, discoverability, legibility, and proposal simulation. More consistent data across the ecosystem is also a prerequisite for future DAO standards.
```

Comment:
For discoverability, consider incorporate [EIP-165](https://eips.ethereum.org/EIPS/eip-165)?

Reply:
Hi @xinbenlv , thanks for the comments! Sorry for the delay in replying; I wanted to have our next working group meeting so I could give others a chance to review some of your suggestions. 

Re: using EIP-165: yes! Thanks for catching this. I believe we had even discussed EIP-165 early on in the meetings but forgot to come back to it before the draft was finalized.

Reply:
BTW, I'll just add we're trying to encourage discussions at https://ethereum-magicians.org/t/erc-4824-decentralized-autonomous-organizations/8362 in case you have additional feedback!